source/clear-linux/network/network-bonding.rst

```diff
@@ -0,0 +1,131 @@
+.. _network-bonding:
+
+Network Bonding
+###############
+
+Network bonding is a technique for combining multiple network interfaces into
+a single, logical interface, providing some combination of redundancy and
+bandwidth aggregation.
+
+|CLOSIA| includes the bonding_ and team_ drivers. Here, we'll provide an
+example of configuring systemd to use the ``bonding`` driver.
+
+First, if it does not already exist, as root, create the
+``/etc/systemd/network`` directory:
+
+.. code-block:: console
+
+   # mkdir -p /etc/systemd/network
+
+In this directory, we'll create the configuration files for the virtual device
+and the network settings for it and the underlying physical interfaces.
+
+Next, we'll configure systemd to create a virtual network device, ``bond1``.
+The syntax for this file is defined in the systemd.netdev_ manpage, and we're
+copying `an example`__ verbatim. Note that ``802.3ad`` mode requires explicit
+support from your NICs and network switch. This and other modes may also
+require additional configuration of your network switch.
+
+__ https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#id-1.20.10
+
+.. code-block:: ini
+   :caption: 30-bond1.netdev
+   :linenos:
+
+   [NetDev]
+   Name=bond1
+   Kind=bond
+
+   [Bond]
+   Mode=802.3ad
+   TransmitHashPolicy=layer3+4
+   MIIMonitorSec=1s
+   LACPTransmitRate=fast
+
+We need to configure the slave interfaces, assigning them to our new
+``bond1``, using the syntax in systemd.network_.
+
+.. code-block:: ini
+   :caption: 30-bond1-enp1s0.network
+   :linenos:
+
+   [Match]
+   Name=enp1s0f*
+
+   [Network]
+   Bond=bond1
+
+   [Link]
+   MTUBytes=9000
+
+.. note::
+
+   I'm bonding all four ports of a quad-port NIC, with names in the range
```

Comment:
You have switched from the first person plural to singular here.

Related to this, is there a convention for these guides to not use the passive voice?